tests/integration/test_global.py

```diff
@@ -726,3 +726,24 @@ def test_global_uninstall_only_reverts_failing(
     assert not tmp_path.joinpath("envs", "dummy-a").is_dir()
     assert dummy_b.is_file()
     assert tmp_path.joinpath("envs", "dummy-b").is_dir()
+
+
+def test_auto_self_expose(pixi: Path, tmp_path: Path, test_data: Path) -> None:
+    env = {"PIXI_HOME": str(tmp_path)}
+    non_self_expose_channel = test_data.joinpath("non_self_expose_channel/output").as_uri()
+
+    # Install dummy from non_self_expose_channel
+    verify_cli_command(
+        [pixi, "global", "install", "--channel", non_self_expose_channel, "non-expose"],
+        ExitCode.SUCCESS,
+        env=env,
+    )
+
+    # Verify that dummy is exposed as 'dummy' through dummy-x as the delivery of the executable.
+    verify_cli_command(
+        [pixi, "global", "list"],
+        ExitCode.SUCCESS,
+        env=env,
+        stdout_contains=["non-expose: 0.1.0"],
+        stdout_excludes=["Nothing"],
+    )
```

Comment:
I feel like asserting that files exists is a bit more readable

Reply:
Jep